rero_ils/modules/documents/jsonschemas/documents/document_contribution_person-v0.0.1.json

```diff
@@ -0,0 +1,107 @@
+{
+  "title": "Person",
+  "type": "object",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "preferred_name",
+    "type",
+    "date_of_birth",
+    "date_of_death",
+    "numeration",
+    "qualifier",
+    "fuller_form_of_name"
+  ],
+  "required": [
+    "type",
+    "preferred_name"
+  ],
+  "properties": {
+    "type": {
+      "title": "Type",
+      "type": "string",
+      "readOnly": true,
+      "default": "bf:Person",
+      "const": "bf:Person",
+      "form": {
+        "templateOptions": {
+          "cssClass": "col-lg-4"
+        }
+      }
+    },
+    "preferred_name": {
+      "title": "Name",
+      "description": "Agent's name.",
+      "type": "string",
+      "minLength": 1,
+      "form": {
+        "placeholder": "Example: M\u00fcller, Hans",
+        "templateOptions": {
+          "cssClass": "col-lg-6"
+        }
+      }
+    },
+    "date_of_birth": {
+      "title": "Birth date",
+      "description": "Record only years.",
+      "type": "string",
+      "minLength": 1,
```

Comment:
can be 4 right?

Reply:
Not if you have books from the romans 400.